src/global.css

```
@@ -8,19 +8,16 @@
   padding: 0;
   box-sizing: border-box;
   font-family: imported;
-  background-color:  	#e9e9e9
 
 }
 
 .container {
   display: flex;
-  max-width: 390px;
+  max-width: 650px;
```

Aqui é bom sempre utilizar o max-width junto com o width para garantir que quando for menor do que 650px ele vai ocupar 100% da tela.

```suggestion
  max-width: 650px;
  width: 100%;
```